### DISCUSSION TITLE
Empty popup on v9.0.1.22

### DISCUSSION THREAD

**Discussion by hydrocanine** [aclap-dev/video-downloadhelper#617] [Help and Q&A]
### Your question

Hi, I'm running Ubuntu 18.04.6 LTS, with Firefox 113.0.2. The VDLH update this morning, 9.0.1.22a1, has stopped working. Clicking on the icon opens an empty box where the media selections would normally be. Image attached.
![vdlh-blankbox](https://github.com/aclap-dev/video-downloadhelper/assets/1882430/1772a0a4-5aff-4e95-9914-90e574d0fc0e)
I can't get to the VDLH settings in its current state, so I can't give the coapp version. It should be the latest.

### Addon & CoApp version and details

I can't get to the VDLH settings in its current state, so I can't give the coapp version. It should be the latest.

**Comment by paulrouget** [comment]
Hey, can you try these instructions: https://github.com/aclap-dev/video-downloadhelper/wiki/How-to-enable-logs and does restart the browser change something?

**Reply by GrampaWildWilly** [reply]
Seems to me I saw something that said VDH requires at least Firefox 115, but I can't find it now.  Am I hallucinating?  In any case, that raises a good point.  The Wiki really should specify minimum levels of operating system version & browser version required by VDH.

**Reply by paulrouget** [reply]
No that's correct. 115 minimum.

**Reply by paulrouget** [reply]
Oh……… damn, the beta doesn't do the check, just the stable. @GrampaWildWilly thanks … that issue really worried me for a minute.

@hydrocanine 2 options: upgrade Firefox, or, install version stable https://addons.mozilla.org/firefox/addon/video-downloadhelper (this will install version 8 of the addon because your firefox is too old)

**Reply by GrampaWildWilly** [reply]
One added step.  Look for the Version History section on that page Paul gave you.  The older versions are in there.

**Comment by hydrocanine** [comment]
Hi,

Finally had a chance to try the suggestions. Trying to run the debugger 
seemed to just hang forever on 'wait...' I did a separate install of the
latest firefox and added video download helper and it seems to work there.
I'll have to do some fiddling to install a later version than firefox 
113 under my ubuntu 18.04 and retain my bookmarks, etc., but, that should
solve my problem.

thanks!
